src/api/app/models/staging/workflow.rb

```diff
@@ -56,13 +56,13 @@ def self.load_groups
     # Group.where(id: group_ids).each do |group|
     #
     # TODO: Refactor this code using to_h when updating to Ruby 2.6 (performance improvement)
-    #
-    groups_hash = {}
-    # FIXME: try to use cache without failure. Like: Rails.cache.fetch(Group.all.cache_key) do
-    Group.find_each do |group|
-      groups_hash[group.title] = group
+    Rails.cache.fetch("groups_hash_#{Group.all.cache_key}") do
```

Comment:
Why is the prefix needed?

Reply:
[@coolo's comment](https://github.com/openSUSE/open-build-service/pull/6815#discussion_r249755960):

> you have to make sure no other code is using this cache_key to cache another return value, so better append some code path specific key

Reply:
I think it doesn't hurt

Reply:
I'm fine with coolo's explanation.

Reply:
note that `['groups_hash', Group.all_cache_key] ` seems to be the stackoverflow preferred way to write this :)